src/construction.jl

```diff
@@ -1,18 +1,24 @@
 "Load all Arrow input data to SubDataFrames that are filtered for used IDs"
 function load_data(config::Dict, starttime::DateTime, endtime::DateTime)
-    node = DataFrame(read_table(config["node"]))
-    edge = DataFrame(read_table(config["edge"]))
-    state = DataFrame(read_table(config["state"]))
-    static = DataFrame(read_table(config["static"]))
-    profile = DataFrame(read_table(config["profile"]))
-    forcing = DataFrame(read_table(config["forcing"]))
+    # Load data and validate schema + rows for required field tyopes and values
+    node = read_table(config["node"]; schema = NodeV1SchemaVersion)
```

Comment:
Is there any reason to pass the type, and not its instance, `NodeV1SchemaVersion()`? The latter feels a bit more natural. And the `schema` keyword can then be set to `::Union{SchemaVersion, Nothing}`.

Reply:
Because Legolas code requires the type, I passed the type. I can make it an instance and use typeof?

Reply:
Ok if Legolas works on the type we can keep it as is, but https://github.com/Deltares/Ribasim.jl/pull/43/files#diff-cd7159638f60b3d2b30ed4bbe7b622ffcdc834f0a2beafe12f46e219a4aa932aR69 (`sv = schema()`) suggests it doesn't.